docs/marketplace/dev/feature-integration-guides/202200.0/merchant-portal-marketplace-customer-specific-prices-feature-integration.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Merchant Portal - Marketplace Customer Specific Prices feature integration
+last_updated: Apr 29, 2022
+description: This document describes the process how to integrate the Marketplace Customer Specific Prices into the Spryker Merchant Portal.
+template: feature-integration-guide-template
+---
+
+This document describes how to integrate the Merchant Portal - Marketplace Customer Specific Prices feature into a Spryker project.
+
+## Install feature core
+
+Follow the steps below to install the Merchant Portal - Marketplace Customer Specific Prices feature core.
+
+### Prerequisites
+
+To start feature integration, integrate the required features:
+
+| NAME | VERSION | INTEGRATION GUIDE                                                                                                                                                 |
+| --------- | ------ |-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Merchant Custom Prices | {{page.version}} | [Merchant Custom Prices feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/merchant-custom-prices-feature-integration.html) |
+| Marketplace Product | {{page.version}} | [Marketplace Product feature integration](/docs/marketplace/dev/feature-integration-guides/{{page.version}}/merchant-portal-marketplace-product-feature-integration.html) |
+
+### 1) Install the required modules using Composer
+
+Install the required modules:
+
+```bash
+composer require spryker-feature/marketplace-merchant-custom-prices --with-dependencies
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following modules were installed:
+
+| MODULE                                            | EXPECTED DIRECTORY        |
+|---------------------------------------------------|---------------------------|
+| PriceProduct                                      | vendor/spryker/price-product |
+| PriceProductMerchantRelationship                  | vendor/spryker/price-product-merchant-relationship |
+| PriceProductMerchantRelationshipMerchantPortalGui | vendor/spryker/price-product-merchant-relationship-merchant-portal-gui |
+| ProductMerchantPortalGui                          | vendor/spryker/product-merchant-portal-gui |
+
+{% endinfo_block %}
+
```

Comment:
The transfer generation section is missing. Please do according to https://spryker.atlassian.net/wiki/spaces/DOCS/pages/478118714/Feature+integration+guides

Reply:
The translation generation section is added as well.